code/renderers/vue3/src/decorateStory.ts

```diff
@@ -30,13 +32,25 @@ function prepare(
       // Normalize so we can always spread an object
       ...normalizeFunctionalComponent(story),
       components: { ...(story.components || {}), story: innerStory },
+      renderTracked(event) {
+        // console.log('innerStory renderTracked', event); // this works only in dev mode
+      },
+      renderTriggered(event) {
+        // console.log('innerStory renderTriggered', event);
+      },
     };
   }
 
   return {
     render() {
       return h(story);
     },
+    renderTracked(event) {
+      // console.log('story renderTracked', event); // this works only in dev mode
```

Comment:
Does this need to be removed?

Reply:
well we could do as i mentioned anyway it runs only in dev mode, it helps other contributor to debug if they need